.github/workflows/integration-test-console.yml

```diff
@@ -385,13 +385,13 @@ jobs:
           make build-latest
           ITMODE_ENABLED=true \
           TRITON_CONDA_ENV_PLATFORM=cpu \
-          RAY_RELEASE_TAG=${RAY_SERVER_VERSION} \
+          RAY_LATEST_TAG=latest \
```

Comment:
We always use `latest`?

Reply:
look so, in `Makefile` and integration-test workflow, any console related test are running with latest backend services

Reply:
Got it. Let's keep it.